Add missing bazel dependencies

Some bazel dependencies (namely `boost`) are not specified in a few places, causing the build to fail on a system without a global installation of `boost`.

e.g.:

```
ERROR: /home/anyong/dev/iotaledger/rpchub/hub/service/BUILD:19:1: C++ compilation of rule '//hub/service:user_address_monitor' failed (Exit 1)
In file included from hub/service/user_address_monitor.cc:15:0:
./hub/db/helper.h:15:10: fatal error: boost/uuid/uuid.hpp: No such file or directory
 #include <boost/uuid/uuid.hpp>
          ^~~~~~~~~~~~~~~~~~~~~
```

--- a/hub/iota/BUILD
+++ b/hub/iota/BUILD
@@ -22,6 +22,11 @@ cc_library(
         "@org_iota_entangled//common/helpers:digest",
         "@org_iota_entangled//common/model:transaction",
         "@com_github_google_glog//:glog",
+        "@boost//:config",
+        "@boost//:core",
+        "@boost//:fusion",
+        "@boost//:move",
+        "@boost//:range"
     ], )
 
 cc_library(

--- a/hub/service/BUILD
+++ b/hub/service/BUILD
@@ -26,6 +26,7 @@ cc_library(
         "@boost//:multi_index",
         "@boost//:range",
         "@boost//:move",
+        "@boost//:uuid"
     ],
     visibility=["//visibility:public"])
 

--- a/hub/stats/BUILD
+++ b/hub/stats/BUILD
@@ -4,5 +4,6 @@ cc_library(
     srcs=glob(["**/*.cc"]),
     deps=[
         "@com_github_google_glog//:glog",
+        "@boost//:uuid"
     ],
     visibility=["//visibility:public"])

